app/src/main/java/org/mozilla/fenix/search/SearchController.kt

```diff
@@ -144,8 +147,9 @@ class DefaultSearchController(
     }
 
     override fun handleExistingSessionSelected(session: Session) {
-        val directions = SearchFragmentDirections.actionSearchFragmentToBrowserFragment(null)
-        navController.nav(R.id.searchFragment, directions)
```

Comment:
I _think_ we should be able to write a lint rule to prevent us from making this mistake.  Out of scope for this bug, but I'm making a note to look into it.